supporting_scripts/analysis-of-endpoint-connections/build.gradle

```diff
@@ -1,6 +1,7 @@
 plugins {
     id 'java'
-    id 'com.github.johnrengelman.shadow' version '7.1.0'
+    id 'application'
+//    id 'com.github.johnrengelman.shadow' version '7.1.0'
```

Comment:
Do we need this comment in there or is it left over from testing/development (lava flow anti-pattern)?

Reply:
It is a left over from testing/development. Thanks for pointing that out.